WFP-VAM/prism-app
# Add an HDC api endpoint to handle chart requests

This fixes issue #791 

How to test the feature:

- Set HDC_TOKEN env in fastapi
- Point the chart_data url to "/hdc" of fastapi server instead of api.earthobservation.vam.wfp.org (changes already included in this PR)

Only warning (not error) will show if the HDC_TOKEN is not set. I saw bug #839, so I thought that should apply to backend as well
